source/darwin/secure_transport_tls_channel_handler.c

```diff
@@ -1096,28 +1096,4 @@ struct aws_tls_ctx *aws_tls_client_ctx_new(struct aws_allocator *alloc, const st
     return s_tls_ctx_new(alloc, options);
 }
 
-void aws_tls_ctx_destroy(struct aws_tls_ctx *ctx) {
```

Comment:
not related to this function. But a duplicates of https://github.com/awslabs/aws-c-io/blob/main/source/darwin/secure_transport_tls_channel_handler.c#L961-L980